### PR TITLE
Update troubleshooting.mdx

### PR DESCRIPTION
**Pull request by jdgemm**
let users know about receive window delay feature if they're having downlink issues...

### CODE REVIEW

**Comment by lthiery** [comment]
There was no mention of the second receive window so I took a fresh pass at explaining this here #773

File: docs/use-the-network/console/troubleshooting.mdx [hunk 1]
```diff
@@ -60,8 +60,9 @@ Remember after initial joining:
...
 - It takes up to (20-30 mins) for a sensor to be added for the first time due to XOR filter adding sensor credentials to blockchain: for more info check [here](https://docs.helium.com/use-the-network/console/adding-devices/#when-adding-devices-for-the-first-time).
 
-<img
-  src={useBaseUrl(
-    "img/use-the-network/console/console-troubleshooting-label-greyed.png"
-  )}
-/>
+### Why aren't downlinks working?
+
+LoRaWAN class A devices require an uplink before they can downlink. According to the LoRaWAN spec a receive window exists for the device to "hear" the downlink.
```
Comment (PharkMillups) [inline docs/use-the-network/console/troubleshooting.mdx]:
```suggestion
LoRaWAN class A devices require an uplink before they can downlink. According to the LoRaWAN spec, a receive window exists for the device to "hear" the downlink. 
```

File: docs/use-the-network/console/troubleshooting.mdx [hunk 2]
```diff
@@ -60,8 +60,9 @@ Remember after initial joining:
...
 
-<img
-  src={useBaseUrl(
-    "img/use-the-network/console/console-troubleshooting-label-greyed.png"
-  )}
-/>
+### Why aren't downlinks working?
+
+LoRaWAN class A devices require an uplink before they can downlink. According to the LoRaWAN spec a receive window exists for the device to "hear" the downlink. 
+If this window (default 1 second) is missed then the downlink will be unsuccessful. With the Receive Window Delay feature users can
```
Comment (PharkMillups) [inline docs/use-the-network/console/troubleshooting.mdx]:
```suggestion
If this window (which defaults to `1` second) is missed then the downlink will be unsuccessful. With the `Receive Window Delay` feature users can 
```

File: docs/use-the-network/console/troubleshooting.mdx [hunk 3]
```diff
@@ -60,8 +60,9 @@ Remember after initial joining:
...
-  src={useBaseUrl(
-    "img/use-the-network/console/console-troubleshooting-label-greyed.png"
-  )}
-/>
+### Why aren't downlinks working?
+
+LoRaWAN class A devices require an uplink before they can downlink. According to the LoRaWAN spec a receive window exists for the device to "hear" the downlink. 
+If this window (default 1 second) is missed then the downlink will be unsuccessful. With the Receive Window Delay feature users can 
+increase the window length and address latency issues that can prevent downlinks. More information about how to use profiles to configure the 
+receive window delay for devices and groups [here](https://docs.helium.com/use-the-network/console/profiles/#receive-window-delay).
```
Comment (PharkMillups) [inline docs/use-the-network/console/troubleshooting.mdx]:
```suggestion
receive window delay for devices and groups [can be found here](https://docs.helium.com/use-the-network/console/profiles/#receive-window-delay).
```

**Review by PharkMillups — COMMENTED** [review]
Left a few small grammar fixes.

**Review by samgutentag — APPROVED** [review]
thank you for your submission!

**Pull request merged**